docs/getting_started/benchmark_owner_demo.md

```diff
@@ -265,10 +267,16 @@ You need to keep at hand the following information:
 {{ demo_url }}
 ```
 
-- The server UIDs of the three MLCubes:
-    - Data preparator UID: `1`
-    - Reference model UID: `2`
-    - Evaluator UID: `3`
+- The server UIDs of the three MLCubes can be found there:
+
+```bash
+ medperf mlcube ls
+```
+
+- For sake of this tutorial predefined UIDs are used: 
+  - Data preparator UID: `1` by default
+  - Reference model UID: `2` by default
+  - Evaluator UID: `3` by default
```

Comment:
```suggestion
- For this tutorial, the UIDs are as follows: 
  - Data preparator UID: `1`
  - Reference model UID: `2`
  - Evaluator UID: `3`
```

Reply:
I've changed this part as these IDs differed for me... Because I made a mistake and submitted cubes in the wrong order:) So I decided to highlight that.
But revisited now, and looks like adding that just complicates the staff, you are right